ddtrace/contrib/requests/patch.py

```diff
@@ -39,8 +40,11 @@ def _traced_request_func(func, instance, args, kwargs):
 
     method = kwargs.get('method') or args[0]
     url = kwargs.get('url') or args[1]
+    parsed_url = urlparse(url)
+    resource = o.path or '/'
```

Comment:
this will cause tons of spans for resources which include ids, timestamps, etc or wildcard domains. this used to be exist but we removed this.